src/main/java/athena/timetable/Timetable.java

```diff
@@ -85,18 +135,229 @@ private void populateTimetable() {
         }
     }
 
+    /**
+     * Generates the timetable header containing hour marks.
+     * For example, +------08------09------10------11------+
+     *
+     * @param startHour The starting hour in 24-hour representation.
+     * @param endHour   The ending hour in 24-hour representation.
+     * @return A string containing the timetable header with hour marks.
+     */
+    private String drawTimetableTimeHeader(int startHour, int endHour) {
```

Comment:
Do you want to shift the methods that print out the Timetable to Ui?

Reply:
At first I was thinking of putting it in `Ui`, and then I feel that eventually we would need to refactor them to somewhere more focussed. Then I realized the somewhere is `Timetable`, since it contains all the related attributes needed to draw the timetable.

Reply:
I think that keeping these this here is better, it keeps calling bits and pieces all the time. If we do eventually do refactor it, the class needs to put the entire timetable in one shot